crates/typst-pdf/src/page.rs

```diff
@@ -245,7 +260,7 @@ pub(crate) fn write_page_labels(ctx: &mut PdfContext) -> Vec<(NonZeroUsize, Ref)
 
 /// Memoized version of [`deflate`] specialized for a page's content stream.
 #[comemo::memoize]
-fn deflate_content(content: &[u8]) -> Arc<Vec<u8>> {
+pub(crate) fn deflate_content(content: &[u8]) -> Arc<Vec<u8>> {
```

Comment:
If this is used in multiple places now, it could be moved to live alongside `fn deflate`. Maybe it should just be called `deflate_memoized`.